### PR TITLE
Attributes support

### PR DESCRIPTION
**Pull request by rithiksharma100**

### CODE REVIEW

File: includes/endpoints/public/class-simpl-checkout-cart-controller.php [hunk 1]
```diff
@@ -7,9 +7,13 @@ function create(WP_REST_Request $request)
             $productID = $request->get_params()["product_id"];
             $variantID = $request->get_params()["variant_id"];
             $quantity = $request->get_params()["quantity"];
-            //TODO: we needs to add validation
+            $variation = array();
```
Comment (Chhekur) [inline includes/endpoints/public/class-simpl-checkout-cart-controller.php:10]: let's be consistent with identifiers naming convention.

Comment (Chhekur) [inline includes/endpoints/public/class-simpl-checkout-cart-controller.php:10]: let's call it `$variation_attr`

File: includes/endpoints/public/class-simpl-checkout-cart-controller.php [hunk 2]
```diff
@@ -7,9 +7,13 @@ function create(WP_REST_Request $request)
...
+            if (isset($request->get_params()["attributes"])) {
+                $variation = $request->get_params()["attributes"];
+            }
+            
             WC()->cart->empty_cart();
-            WC()->cart->add_to_cart($productID, $quantity, $variantID);
+            WC()->cart->add_to_cart($productID, $quantity, $variantID, $variation);
```
Comment (Chhekur) [inline includes/endpoints/public/class-simpl-checkout-cart-controller.php:16]: same^

File: includes/endpoints/public/class-simpl-checkout-cart-controller.php (hunk 1 above)
Comment (rithiksharma100) [reply includes/endpoints/public/class-simpl-checkout-cart-controller.php:10]: done

Comment (rithiksharma100) [reply includes/endpoints/public/class-simpl-checkout-cart-controller.php:10]: done

File: includes/endpoints/public/class-simpl-checkout-cart-controller.php (hunk 2 above)
Comment (rithiksharma100) [reply includes/endpoints/public/class-simpl-checkout-cart-controller.php:16]: done

**Review by Chhekur — APPROVED** [review]
LGTM!

**Pull request merged**